### ISSUE TITLE
Alternative Version of Site with Improved Accessibility and Contributor Attribution

### ISSUE THREAD

**Issue by calebfoss** [processing/p5.js-website#689]
I teach at a community college, and I wanted my students to have access to the new p5 site's content, but I have accessibility and attribution concerns about the site.

I deployed an alternative version of the new p5 site at [p5-access.vercel.app](https://p5-access.vercel.app/) with improved accessibility and also setting a better example for my students of attribution for others' code. I wanted to share this with the community for anyone else that may benefit.

I implemented the following changes:
- Remove keyboard traps in code editors so that people with disabilities that use keyboard navigation may use the site. (#666)
- Adjust colors to fix contrast issues that fail WCAG standards. (#680 / #687 and #688)
- Add names of contributors who worked on examples. I manually checked that all contributors who had been individually attributed on the old site are attributed as well as those who were attributed on the Processing site for examples that were ported to p5. I also created a build script to query the GitHub API, pull additional authorship information for examples, and add those authors. This means that updating attribution can be performed by simply running the script. (#415)
- Restore the git history for example files. This also resolves #437.
- Link previous versions of examples from the old site and/or Processing where applicable to meet requirements of the [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/#ref-indicate-changes) license.

It concerns and disappoints me that the Processing Foundation accepted [€250,000 in funds](https://www.sovereign.tech/tech/p5js) from @sovereigntech to develop the new site with an "accessibility-first approach" and then launched a site that introduces major new accessibility issues that were not present on the old site and removes contributor attribution that was present on the old site. I hope that the organization will take accountability, realign with their values, and make things right. If they do, they are welcome to use [my fork](https://github.com/calebfoss/p5.js-website) as a reference for changes.
